hooks/useUser.ts

```diff
@@ -209,35 +232,69 @@ const useUser = () => {
   }
 
   const safeAA = useCallback(async (passkey: PasskeyArgType) => {
-    return Safe4337Pack.init({
-      provider: rpcUrls[mainnet.id],
-      signer: passkey,
-      bundlerUrl: USER.pimlicoUrl,
-      // paymasterOptions: {
-      //   isSponsored: true,
-      //   paymasterUrl: USER.pimlicoUrl
-      // },
-      options: {
-        owners: [],
-        threshold: 1
-      }
-    })
-  }, [])
+    const { x, y, prefix } = PublicKey.from({
+      prefix: 4,
+      x: BigInt(passkey.coordinates.x),
+      y: BigInt(passkey.coordinates.y),
+    });
+    const webauthnValidator = getWebAuthnValidator({
+      pubKey: { x, y, prefix },
+      authenticatorId: passkey.credentialId,
+    });
 
-  const userOpReceipt = useCallback(async (safe4337Pack: Safe4337Pack, userOperationHash: string) => {
-    let userOperationReceipt = null
-    while (!userOperationReceipt) {
-      await new Promise((resolve) => setTimeout(resolve, 2000))
-      userOperationReceipt = await safe4337Pack.getUserOperationReceipt(
-        userOperationHash
-      )
-    }
-    return userOperationReceipt
+    const deadOwner = toAccount({
+      address: "0x000000000000000000000000000000000000dead",
+      async signMessage() {
+        return "0x";
+      },
+      async signTransaction() {
+        return "0x";
+      },
+      async signTypedData() {
+        return "0x";
+      },
+    });
+
+    const safeAccount = await toSafeSmartAccount({
+      saltNonce: getNonce({
+        appId: 'solid',
+      }),
+      client: publicClient(mainnet.id),
+      owners: [deadOwner],
```

Comment:
@LiorAgnin is item in owners array required? we can keep it empty:
`owners: []`

Reply:
@MusabShakeel576  Looks like it is required
https://docs.pimlico.io/references/permissionless/reference/accounts/toSafeSmartAccount#owners